### ISSUE TITLE
packet completion is missing in the Windows binary 

### ISSUE THREAD

**Issue by woa7** [equinix/metal-cli#67]
### What happened?

using packet-cli in a cygwin bash Session,
packet completion bash
dont work:
 root@box ~
$ packet completion -h
Error: unknown command "completion" for "packet"
Run 'packet --help' for usage.

root@box ~
$ packet completion bash
Error: unknown command "completion" for "packet"
Run 'packet --help' for usage.
root@box ~
$

CLI version (`packet --version`):
$ packet --version
packet version 0.0.7

**Comment by displague** [comment]
@woa7 The completions subcommand has not been included in a tagged release yet.

To give it a try:

```
git clone https://github.com/packethost/packet-cli
cd packet-cli
make install # assumes Go is installed
eval "$(packet completion bash)"
```

**Comment by displague** [comment]
@woa7 I just pushed the v0.0.8 release with the completions support.

Let me know if the Windows binary behaves as you would expect:
https://github.com/packethost/packet-cli/releases/tag/0.0.8

**Comment by MaxPeal** [comment]
@displague ok with the v0.0.8 release its half working.

bugs:
- the error message output ist not stderr!
- the check for PACKET_TOKEN is rong, as so "source <(packet completion bash)" is not working
- the "packet completion" is not listet in packet --help

again using packet-cli in a cygwin bash Session:
```
root@box ~
$ packet --version
packet version 0.0.8

root@box ~
$ packet completion -h
To load completions:

        Bash:

        $ source <(packet completion bash)

        Bash (3.2.x):

        $ eval "$(packet completion bash)"

        # To load completions for each session, execute once:
        Linux:
          $ packet completion bash > /etc/bash_completion.d/packet-cli
        MacOS:
          $ packet completion bash > /usr/local/etc/bash_completion.d/packet-cli

        Zsh:

        $ source <(packet completion zsh)

        # To load completions for each session, execute once:
        $ packet completion zsh > "${fpath[1]}/_packet-cli"

        Fish:

        $ packet completion fish | source

        # To load completions for each session, execute once:
        $ packet completion fish > ~/.config/fish/completions/packet-cli.fish

Usage:
  packet completion [bash|zsh|fish|powershell]

Flags:
  -h, --help   help for completion

Global Flags:
      --config string   Path to JSON or YAML configuration file
```
```
root@box ~
$ packet -h
Command line interface for Packet Host

Usage:
  packet [command]

Available Commands:
  2fa                  Two Factor Authentication operations
  capacity             Capacities operations
  device               Device operations
  event                Events operations
  facilities           Facility operations
  hardware-reservation Hardware reservation operations
  help                 Help about any command
  ip                   IP operations
  operating-systems    Operating system operations
  organization         Organization operations
  plan                 Plan operations
  project              Project operations
  ssh-key              SSH key operations
  user                 User operations
  virtual-network      Virtual network operations
  volume               Volume operations
  vpn                  VPN service operations

Flags:
      --config string   Path to JSON or YAML configuration file
  -h, --help            help for packet
  -v, --version         version for packet

Use "packet [command] --help" for more information about a command.
```
```
root@box ~
$ source <(packet completion bash)
Error: unknown command "authentication" for "packet"
Run 'packet --help' for usage.
```
```
root@box ~
$ set -vx

root@box ~
$ source <(packet completion bash)
source <(packet completion bash)
+ source /dev/fd/63
packet completion bash
++ packet completion bash
Packet authentication token not provided. Please either set the 'PACKET_TOKEN' environment variable or create a JSON or YAML configuration file.
++ Packet authentication token not provided. Please either set the PACKET_TOKEN environment variable or create a JSON or YAML configuration file.
Error: unknown command "authentication" for "packet"
Run 'packet --help' for usage.
```
```
root@box ~
$ eval "$(packet completion bash)"
eval "$(packet completion bash)"
++ packet completion bash
+ eval 'Packet authentication token not provided. Please either set the '\''PACKET_TOKEN'\'' environment variable or create a JSON or YAML configuration file.'
Packet authentication token not provided. Please either set the 'PACKET_TOKEN' environment variable or create a JSON or YAML configuration file.
++ Packet authentication token not provided. Please either set the PACKET_TOKEN environment variable or create a JSON or YAML configuration file.
Error: unknown command "authentication" for "packet"
Run 'packet --help' for usage.
```
```
root@box ~
$ packet completion bash | head
packet completion bash | head
+ packet completion bash
+ head
Packet authentication token not provided. Please either set the 'PACKET_TOKEN' environment variable or create a JSON or YAML configuration file.
```
```
root@box ~
$ PACKET_TOKEN=fooo packet completion bash | head
PACKET_TOKEN=fooo packet completion bash | head
+ PACKET_TOKEN=fooo
+ packet completion bash
+ head
# bash completion for packet                               -*- shell-script -*-

__packet_debug()
{
    if [[ -n ${BASH_COMP_DEBUG_FILE} ]]; then
        echo "$*" >> "${BASH_COMP_DEBUG_FILE}"
    fi
}
```

# Homebrew on Macs have version 1.3 of bash-completion which doesn't include

root@box ~
$`

**Comment by displague** [comment]
Thanks for all of these details, @MaxPeal! (I touched up your comment with triple-backticks for readability)

> the error message output ist not stderr!

This is captured by https://github.com/packethost/packet-cli/issues/75 (I added your additional notes).

> the check for PACKET_TOKEN is wrong, as so "source <(packet completion bash)" is not working

This should be resolved in the `master` branch, via #74 

> the "packet completion" is not listet in packet --help

This was a design choice that I thought no one would mind and I now regret.
I'll create a new issue to reflect this.
